pep-0214.txt

```diff
@@ -5,366 +5,384 @@ Last-Modified: $Date$
-
-    I'm not 100% sure that the choice for a statement was right (ABC
-    had the compelling reason that it used statement syntax for
-    anything with side effects, but Python doesn't have this
-    convention), but I'm also not convinced that it's wrong.  I
-    certainly like the economy of the print statement.  (I'm a rabid
-    Lisp-hater -- syntax-wise, not semantics-wise! -- and excessive
-    parentheses in syntax annoy me.  Don't ever write return(i) or
-    if(x==y): in your Python code! :-)
-
-    Anyway, I'm not ready to deprecate the print statement, and over
-    the years we've had many requests for an option to specify the
-    file.
-
-    Challenge: Why not > instead of >>?
-
-    Response: To DOS and Unix users, >> suggests "append", while >
-    suggests "overwrite"; the semantics are closest to append.  Also,
-    for C++ programmers, >> and << are I/O operators.
-
-    Challenge: But in C++, >> is input and << is output!
-
-    Response: doesn't matter; C++ clearly took it from Unix and
-    reversed the arrows.  The important thing is that for output, the
-    arrow points to the file.
-
-    Challenge: Surely you can design a println() function can do all
-    what print>>file can do; why isn't that enough?
-
-    Response: I think of this in terms of a simple programming
-    exercise.  Suppose a beginning programmer is asked to write a
-    function that prints the tables of multiplication.  A reasonable
-    solution is:
-
-        def tables(n):
-            for j in range(1, n+1):
-                for i in range(1, n+1):
-                    print i, 'x', j, '=', i*j
-                print
-
-    Now suppose the second exercise is to add printing to a different
-    file.  With the new syntax, the programmer only needs to learn one
-    new thing: print >> file, and the answer can be like this:
-
-        def tables(n, file=sys.stdout):
-            for j in range(1, n+1):
-                for i in range(1, n+1):
-                    print >> file, i, 'x', j, '=', i*j
-                print >> file
-
-    With only a print statement and a println() function, the
-    programmer first has to learn about println(), transforming the
-    original program to using println():
-
-        def tables(n):
-            for j in range(1, n+1):
-                for i in range(1, n+1):
-                    println(i, 'x', j, '=', i*j)
-                println()
-
-    and *then* about the file keyword argument:
-
-        def tables(n, file=sys.stdout):
-            for j in range(1, n+1):
-                for i in range(1, n+1):
-                    println(i, 'x', j, '=', i*j, file=sys.stdout)
-                println(file=sys.stdout)
+  vs.
 
-    Thus, the transformation path is longer:
+  ::
 
-        (1) print
-        (2) print >> file
+      (1) print
+      (2) println()
+      (3) println(file=...)
 
-    vs.
+  Note: defaulting the file argument to sys.stdout at compile time
```

Comment:
You've changed sys.stdout to \`\`sys.stdout\`\` in line 339. Why did you keep this one unformatted?